flow-preferences/src/main/java/com/fredporciuncula/flow/preferences/FlowSharedPreferences.kt

```diff
@@ -4,70 +4,154 @@ import android.content.SharedPreferences
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.flow.Flow
 import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
 
 internal typealias KeyFlow = Flow<String?>
 
+/**
+ * Specific wrapper, that presents wide range of methods for handful using of [SharedPreferences] via [Preference] wrapper
+ * @param sharedPreferences takes an instance of [SharedPreferences]
+ * @param coroutineContext takes a [CoroutineContext]. Can be used to override standard behavior, e.g. logging
+ */
 class FlowSharedPreferences @JvmOverloads constructor(
   val sharedPreferences: SharedPreferences,
-  val coroutineContext: CoroutineContext = Dispatchers.IO
+  val coroutineContext: CoroutineContext = EmptyCoroutineContext,
```

Comment:
Yes, by default we want to always do I/O on dispaters.io. In this variant, it will be so if you do not redefine the dispatcher. In addition, debugging often requires naming the coroutine or passing other context elements. This is the recommendation of Google and JetBrains and in my experience, it is quite justified.

Reply:
I think I would still prefer to stick with `Dispatchers.IO` by default instead of an `EmptyCoroutineContext`. This also matches the documentation written for the `FlowSharedPreferences` where we're mentioning the `Dispatchers.IO`.

Reply:
Okay, I will revert this change